Assets/Scripts/IObstacles.cs

```diff
@@ -0,0 +1,6 @@
+
+public interface IObstacles
+{
+    public void MoveObstacle();
```

Comment:
non-blocking: Because it's under `IObstacle`, maybe this doesn't need to repeat what it moves. Is the `Move` in use from somewhere? if not, Can it be updated to `Move`?